cmd/juju/commands/bootstrap.go

```diff
@@ -107,7 +111,10 @@ Examples:
 See also:
     add-credentials
     add-model
-    set-constraints`
+    set-constraints
+    controller-config --help
```

Comment:
We don't need the help here - all commands have --help, and See also typically refers just to the command names.
If you did  want to mention the commands with --help, it could be done above in the prose used when printing the doc URLs.